LoRaEngine/LoraKeysManagerFacade/LoRaRegistryManager.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoraKeysManagerFacade
+{
+    using System;
+    using Microsoft.Azure.Devices;
+    using Microsoft.Extensions.Configuration;
+
+    public static class LoRaRegistryManager
+    {
+        private static readonly object SingletonLock = new object();
+
+        private static RegistryManager current;
+
+        /// <summary>
+        /// Explicit initialization of a <see cref="RegistryManager"/> implementation.
+        /// </summary>
+        /// <param name="testManager">RegistryManager to be used</param>
+        /// <remarks>Note: only used for unit testing</remarks>
+        public static void InitRegistryManager(RegistryManager testManager)
```

Comment:
I think you mean `registryManager` instead of `testManager`

Reply:
I only want this to be used in unit tests, that's why I named it this way to make it more obvious.